src/isomorphic/React.js

```diff
@@ -59,8 +59,9 @@ if (__DEV__) {
   createMixin = function(mixin) {
     lowPriorityWarning(
       warnedForCreateMixin,
-      'React.createMixin is deprecated and should not be used. You ' +
-        'can use this mixin directly instead.',
+      'In React 16.0, React.createMixin is deprecated and should not be used. ' +
+        'You can use this mixin directly instead.' +
+        ' See (https://fb.me/createmixin-was-never-implemented) for more info.',
```

Comment:
Yea, but looking at it again I don't think we need the parens. Will fix, thanks for pointing it out. :)